NSS-Day-Cohort-49/giffygram-success-kids
# Ar footer and profiles

#### Changes Made
1. added minor css to the postEntry in order to resize the description box and give some margin space to the whole form.
2. attempted edit button (didn't work but maybe next time). 
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout branchname
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: When user uses the postEntry form, it should be spaced out and sized correctly rather than cramped up.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
